### ISSUE TITLE
`chrome` is not defined / widevinecdm is out of date

### ISSUE THREAD

**Issue by jackquack** [jiahaog/nativefier#68]
I experienced that error in the console when I tried to watch something w/ Netflix. 

![screen shot 2016-01-23 at 12 03 53 pm](https://cloud.githubusercontent.com/assets/554881/12531571/09c4a4c2-c1cc-11e5-8ed5-753b4adbb93f.png)

It's possible that if I could open a tab and follow the instructions that Netflix is suggesting that my problem could be solved...

**Comment by jiahaog** [comment]
If you access this page from your default chrome browser, does it require flash?

**Comment by jackquack** [comment]
I think that Netflix uses Silverlight actually...

**Comment by neilco** [comment]
Netflix in Chrome uses the HTML5 video player. 

<img width="1253" alt="screen shot 2016-01-25 at 08 47 30" src="https://cloud.githubusercontent.com/assets/59509/12546227/4fa24f2e-c340-11e5-9caf-58a16fece5c3.png">

**Comment by jiahaog** [comment]
I don't have a netflix account, are there any other websites that I can test this on? It should work if it uses the HTML5 video player though :(

**Comment by jackquack** [comment]
From Googling, it's hard to find out what the present situation is with them. Historically it looks like you had the option to use HTML5 OR silverlight. I'm not sure what it was trying to serve me. That widevinecdm error relates to something called the Widevine Content Decryption Module. I'm not sure what that is though...

**Comment by jiahaog** [comment]
Consolidating discussion under #113

**Comment by quanglam2807** [comment]
@hrdwdmrbl @neilco It has been fixed in WebCatalog webcatalog/desktop#4

**Comment by mikhoul** [comment]
I'm new with nativefier and I had the same problem with  Spotify, browsing the issue **I think it was fixed** but is have to be approved before going in the main branch: https://github.com/jiahaog/nativefier/pull/440 

Sorry I'm a newbie with github technical terms,

Regards :octocat:

**Comment by xdgc** [comment]
PR #440 looks promising, but it didn't work for me. It's possible that my whole nativefier installation is tainted because I installed using Homebrew first, then tried to install from git.

But still, I have a working Netflix app. The basic guidelines on how to enable widevine cdm are accurate, but the hitch is that you have to point it to a widevine binary that's valid for not only for your platform, but also for the Chrome runtime _in the electron app_. That means you can't simply point it at your "real" Chrome browser.

To start, I built a Netflix app that didn't work. I opened developer tools to get a JS console, so I could obtain the exact Chrome runtime version.

```
navigator.userAgent
```

In my nativefier/electron combo, that gives 56.0.2924.87. So the trick is to get a Widevine bin from that release of Chrome. I went to https://google-chrome.en.uptodown.com/ to find that. The nearest match was 56.0.2924.76, which is fine. I unpacked that to a temporary location.

I created a new folder, `/Applications/Google Chrome Electron Support/56.0.2924.76`. Then I copied my temporary `Google Chrome.app/Contents/Versions/56.0.2924.76/Google Chrome Framework.framework/Libraries/WidevineCdm/_platform_specific/mac_x64/*` to that folder. This obviously is for a Mac but the procedure for anything else should be similar.

Finally I went to my nativefied Netflix.app and edited `Netflix.app/Contents/Resources/app/lib/main.js`. Right before the bits about `setDockBadge` I added these lines:

```
    /// add widevine DRM support
    _electron.app.commandLine.appendSwitch('widevine-cdm-path', '/Applications/Google Chrome Electron Support/56.0.2924.76/widevinecdmadapter.plugin');
    _electron.app.commandLine.appendSwitch('widevine-cdm-version', '1.4.8.962');
```

I got the widevine version from running `strings widevinecdmadapter.plugin | grep '1\.'.  This way I don't have to worry about an older Chrome runtime messing anything up in my user profile.

You can make those changes in nativefier's main.js file to effect the support for any new app.

Again I think the new PR shows some promise — I'm going to scrub down as much as I can and give it another go — but I wanted to get this report out here while it's fresh in mind.
